openid-4-verifiable-presentations-1_0.md

```diff
@@ -611,6 +629,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+The attachment data mechanism enables to pass additional information to the Wallet to request credentials. The Verifier MAY include attachments in the Authorization Request, for example to pass a registration certificate to inform about the registered intended use or an authorization attestation to request a specific credential from the Wallet to comply with the embedded disclosure policy.
+
+### Proof of Possession
+
+In some cases, the Verifier may want to include attachments that are bound to a key or require verification of their origin.
+
+It is up to the ecosystem profile to define how proof of possession is conveyed. Verifiers SHOULD use the `nonce` and `client_id` parameter from the Authorization Request in any proof-of-possession mechanism to bind to this transaction. The Wallet MUST validate such proofs if defined by the profile and ignore or reject attachments that fail validation.
+
+### Multi-Relying Party Behavior
+
+In some cases, the Verifier may want to include attachments based on specific ecosystem profiles bound to specific Client Identifier Prefixes. In such a scenario, the Authorization Request MUST sent a signed request in the JWS JSON Serialization format, see (#jws-json-serialization).
```

Comment:
Why don't you just add this to the multi RP section in the DC API Annex? Then the context would be cristal clear.

Reply:
From a readers perspective this would be the first place for me to search for it (like the specific behavior when adding a feature. But we can move it down.

Reply:
Removed this section